components/button/demo/button.html

```diff
@@ -21,27 +21,12 @@ <h2>Button</h2>
 				</template>
 			</d2l-demo-snippet>
 
-			<h2>Button (Primary)</h2>
 
-			<d2l-demo-snippet>
-				<template>
-					<d2l-button primary>Primary Button</d2l-button>
-				</template>
-			</d2l-demo-snippet>
-
-			<h2>Button (Disabled)</h2>
-
-			<d2l-demo-snippet>
-				<template>
-					<d2l-button disabled>Disabled Button</d2l-button>
-				</template>
-			</d2l-demo-snippet>
-
-			<h2>Button (Disabled with Tooltip)</h2>
+			<h2>Toggle me! (Disabled with Tooltip)</h2>
 
 			<d2l-demo-snippet>
 				<template>
-					<d2l-button disabled disabled-tooltip="Optional disabled tooltip">Normal Button</d2l-button>
+					<d2l-button id="toggle" disabled-tooltip="Optional disabled tooltip">Normal Button</d2l-button>
```

Comment:
The disabled attribute is being removed from this demo, is that expected?

Reply:
Ok, let's be sure to clean this (and the script below) up before merging. Or, since this is a behaviour that is more general to tooltips, it might be nice to have an actual demo in the tooltip demo page - if you do, I'd put the script into the template so the person reading the demo code can understand. That also avoids timing issues with trying to wire up event handlers before the demos are stamped.